### PR TITLE
release notes for v6.0.6-24

### PR DESCRIPTION
**Pull request by alexmilowski**
Added release notes for version 6.0.6-24 and changed 6.0.6-23 notes to not say "latest".

### CODE REVIEW

**Review by amiramm — APPROVED** [review]
LGTM, thanks @alexmilowski

File: content/platforms/release-notes/k8s-6-0-6-24-2020-08.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Openshift 3.11 doesn't support dockerhub private registry. This is a known OpenShift issue.
+
+### Internal DNS and Kubernetes DNS may have conflicts (RED37462)
+
+DNS conflicts are possible between the cluster mdns_server and the K8s DNS. This only impacts DNS resolution from within cluster nodes for Kubernetes DNS names.
+
+### 5.4.10 negatively impacts 5.4.6 (RED37233)
+
+Kubernetes-based 5.4.10 clusters seem to negatively impact existing 5.4.6 clusters.
```
Comment (bmansheim) [inline content/platforms/release-notes/k8s-6-0-6-24-2020-08.md:69]: @alexmilowski It seemed that the word clusters was missing. Correct?

Comment (amiramm) [reply content/platforms/release-notes/k8s-6-0-6-24-2020-08.md:69]:
```suggestion
Kubernetes-based 5.4.10 deployments seem to negatively impact existing 5.4.6 deployments that share a Kubernetes cluster.
```
@bmansheim better?

**Review by amiramm — COMMENTED** [review]
LGTM

**Pull request merged**